cli/test/e2e/login-key.e2e-spec.ts

```diff
@@ -28,6 +32,12 @@ describe(`login-key (e2e)`, () => {
 
     const api = await setup();
     apiKey = api.apiKey;
+
+    await unlink(TEST_AUTH_FILE).catch(() => {});
+  });
+
+  afterEach(async () => {
+    await unlink(TEST_AUTH_FILE).catch(() => {});
```

Comment:
Probably just me being confused but why do we need to unlink the file at the beginning and the end of each test? Isn't beginning enough?

Reply:
This is just to tidy up for the next test suite. Maybe I can move it to the afterAll instead

Reply:
done